### PR TITLE
Add data model PR

### PR DESCRIPTION
**Pull request by diarisdiakite**
In this PR, I:
📃 Created a new app called Blog-app.
📃 Initialized your project with Git.
📃 Mdke sure that your project has Postgres database set up.
📃 Built the project schema.
✔️ Created and ran the necessary migration files.
✔️ Table and column names match the ERD diagram.
✔️ Photo for users table is a link to an image
✔️ Foreign keys are included.
✔️ In posts table the column holding the foreign key to the users table is named author_id
✔️ All columns that are foreign keys have a corresponding index.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (iambenkis) [inline README.md:205]: - [ ] Good job creating your README file , however to keep it professional you should avoid broken link 👍 . A broken link is attached to your MIT link 😄 

Capture

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @diarisdiakite 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️ Schema created according to [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png)

## Suggested changes

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (diarisdiakite) [reply README.md:205]: Done @iambenkis, MIT licence added. Thanks!

**Review by cynthiainga — APPROVED** [review]
Hi @diarisdiakite, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] Requested change is made ✔️ 
- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**